transcoder/message/handler/SequencerHandler.py

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2022 Google LLC
```

Comment:
2023

Reply:
RiFCO?

Reply:
https://github.com/github/site-policy/blob/1263bf0164cfe60d0900e70ba9126855c7c23c6c/Policies/github-terms-of-service.md

Reply:
changed in latest